### PR TITLE
Updating Windows intro and resource management pages

### PR DESCRIPTION
**Pull request by marosset**
- Fixing heading levels for /docs/concepts/windows/intro.md
- Misc updates to /docs/concepts/windows/intro.ms
- Updates to /docs/concepts/configuration/windows-resource-management.md
for accuracy

Signed-off-by: Mark Rossetti <marosset@microsoft.com>

Part of https://github.com/kubernetes/website/issues/31428
Fixes some comments in https://github.com/kubernetes/website/pull/33582

### CODE REVIEW

File: content/en/docs/concepts/windows/intro.md [hunk 1]
```diff
@@ -260,17 +260,18 @@ The following list documents differences between how Pod specifications work bet
...
 None of the Pod [`securityContext`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context) fields work on Windows.
 
-### Node problem detector
+## Node problem detector
 
 The node problem detector (see
 [Monitor Node Health](/docs/tasks/debug/debug-cluster/monitor-node-health/))
-is not compatible with Windows.
+has prelimiary support for Windows.
+For more information visit the the project's [GitHub page](https://github.com/kubernetes/node-problem-detector#windows)
```
Comment (tengqm) [inline content/en/docs/concepts/windows/intro.md:272]:
```suggestion
For more information, visit the project's [GitHub page](https://github.com/kubernetes/node-problem-detector#windows).
```

File: content/en/docs/concepts/windows/intro.md [hunk 2]
```diff
@@ -375,7 +374,7 @@ explains how to deploy Windows nodes to your cluster using kubeadm.
 
 The Kubernetes [cluster API](https://cluster-api.sigs.k8s.io/) project also provides means to automate deployment of Windows nodes.
 
-### Windows distribution channels
+## Windows distribution channels
 
 For a detailed explanation of Windows distribution channels see the [Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19).
```
Comment (tengqm) [inline content/en/docs/concepts/windows/intro.md:379]:
```suggestion
For a detailed explanation of Windows distribution channels see the
[Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19).
```

File: content/en/docs/concepts/windows/intro.md [hunk 3]
```diff
@@ -260,17 +260,18 @@ The following list documents differences between how Pod specifications work bet
...
 
 None of the Pod [`securityContext`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context) fields work on Windows.
 
-### Node problem detector
+## Node problem detector
 
 The node problem detector (see
 [Monitor Node Health](/docs/tasks/debug/debug-cluster/monitor-node-health/))
-is not compatible with Windows.
+has prelimiary support for Windows.
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/windows/intro.md:271]:
```suggestion
has preliminary support for Windows.
```

File: content/en/docs/concepts/windows/intro.md [hunk 4]
```diff
@@ -318,7 +319,7 @@ when using GMSA with containerd to access Windows network shares, which requires
 kernel patch.
 {{< /note >}}
 
-#### Mirantis Container Runtime {#mcr}
+### Mirantis Container Runtime {#mcr}
 
 [Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR) is available as a container runtime for all Windows Server 2019 and later versions.
```
[the remark this answers is not in the document]
Comment (divya-mohan0209) [reply content/en/docs/concepts/windows/intro.md:324]:
```suggestion
[Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR)
is available as a container runtime for Windows Server 2019 and all later versions of Windows.
```

File: content/en/docs/concepts/windows/intro.md [hunk 5]
```diff
@@ -364,9 +365,7 @@ reported previously and comment with your experience on the issue and add additi
 logs. SIG-Windows Slack is also a great avenue to get some initial support and
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/windows/intro.md:365]:
```suggestion
logs. SIG-Windows channel on the Kubernetes Slack is also a great avenue to get some initial support and
```

**Review by divya-mohan0209 — COMMENTED** [review]
few grammatical nits

File: content/en/docs/concepts/configuration/windows-resource-management.md [hunk 6]
```diff
@@ -32,52 +32,47 @@ host, and thus privileged containers are not available on Windows.
...
-On Windows, the kubelet supports a command-line flag to set the priority of the
+On Windows, the kubelet supports a command-line flag to set the
+[scheduling priority](https://docs.microsoft.com/windows/win32/procthread/scheduling-priorities) of the
 kubelet process: `--windows-priorityclass`. This flag allows the kubelet process to get
 more CPU time slices when compared to other processes running on the Windows host.
 More information on the allowable values and their meaning is available at
 [Windows Priority Classes](https://docs.microsoft.com/en-us/windows/win32/procthread/scheduling-priorities#priority-class).
 To ensure that running Pods do not starve the kubelet of CPU cycles, set this flag to `ABOVE_NORMAL_PRIORITY_CLASS` or above.
+
+## Resource Reservation {$resource-reservation}
```
Comment (sftim) [inline content/en/docs/concepts/configuration/windows-resource-management.md:59]:
```suggestion
## Resource reservation {#resource-reservation}
```

File: content/en/docs/concepts/windows/intro.md [hunk 7]
```diff
@@ -50,7 +50,8 @@ Some node features are only available if you use a specific
 including:
 
 * HugePages: not supported for Windows containers
-* Privileged containers: not supported for Windows containers
+* Privileged containers: not supported for Windows containers.
+  [HostProcess Containers](/docs/tasks/configure-pod-container/create-hostprocess-pod.md) offer similar functionality.
```
Comment (sftim) [inline content/en/docs/concepts/windows/intro.md:54]:
```suggestion
  [HostProcess Containers](/docs/tasks/configure-pod-container/create-hostprocess-pod/) offer similar functionality.
```

---
_aside_
Ideally, we'd add a heading about HostProcess containers into the https://kubernetes.io/docs/concepts/containers/ section overview. This link could then be to that heading.

However, that might first mean adding a section into https://kubernetes.io/docs/concepts/containers/  about supported OSs for containers, which I think is best done in its own PR.

File: content/en/docs/concepts/windows/intro.md (hunk 5 above)
Comment (divya-mohan0209) [reply content/en/docs/concepts/windows/intro.md:365]: Thank you for the catch, Tim!
